### ISSUE TITLE
Any plans for adding feature of commenting on papers? 

### ISSUE THREAD

**Issue by andreas-bulling** [karpathy/arxiv-sanity-preserver#7]
Would be great, e.g. for research groups to discuss papers/add comments/thoughts/etc

**Comment by karpathy** [comment]
yes, I think more generally a social layer overlay over the site could be interesting and is on my mind as a potential feature going forward.

**Comment by andreas-bulling** [comment]
Didn't think of something particularly complex. General comments could be good first step, next step could be to implement something similar to http://theoj.org/

**Comment by andreas-bulling** [comment]
Let's see - I'll try to get started on the comment functionality if I find some time at the weekend.

**Comment by karpathy** [comment]
I would encourage you not to :) I think I have to do some sweeping changes in the guts of the project because it's not laid out in a way that makes it easy to add recent papers. I suspect I'll have to change quite a few things. Also now that I decided I want this project to be legitimate I'd like to make sure the scripts are not quick hacks but something that flows. In other words, major changes coming soon.

**Comment by andreas-bulling** [comment]
Alright ;)

**Comment by MartinThoma** [comment]
For the comments, https://disqus.com/ might be the easiest way.
